babel/src/main/codegen/config.fmpp

```diff
@@ -32,6 +32,7 @@ data: {
       "IF"
       "SEMI"
       "VOLATILE"
+      "SEL"
```

Comment:
nit: let's keep these in alphabetical order.

Reply:
Just wanted to hop in and inquire if putting these in reserved keywords instead of non-reserved could have any issues down the line? For example, what if a user in a dialect that doesn't have SEL uses it for an attribute name or something?

Reply:
I changed the order of the keywords, but I am also wondering now if we should put it as a non-reserved keyword since only some dialects use it

Reply:
Per our conversation today, nonReservedKeywords SGTM

Reply:
Made a commit to move SEL to nonReservedKeywords